src/signals/incident/components/form/ContainerSelect/Selector/SelectionPanel/SelectionPanel.tsx

```diff
@@ -0,0 +1,146 @@
+  margin-bottom: 0;
+  font-size: 16px;
+  opacity: 0.6;
+`;
+
+const EmptySelectionWrapper = styled.div`
+  background-color: ${themeColor('tint', 'level2')};
+  height: 100px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  margin: ${themeSpacing(4)} 0;
+`;
+
+const StyledButton = styled(Button)`
+  margin-top: ${themeSpacing(6)};
+`;
+
+export interface SelectionPanelProps {
+  onChange: (items: Item[]) => void;
+  onClose: () => void;
+  variant: Variant;
+  selection: Item[];
+  featureTypes: FeatureType[];
+}
+
+const SelectionPanel: FunctionComponent<SelectionPanelProps> = ({
+  onChange,
+  onClose,
+  variant,
+  selection,
+  featureTypes,
+}) => {
+  const selectionOnMap = useMemo(
+    () => selection.filter(container => container.type !== CONTAINER_NOT_ON_MAP_TYPE_NAME),
+    [selection]
+  );
+  const notOnMapContainer = useMemo(
+    () => selection.find(container => container.type === CONTAINER_NOT_ON_MAP_TYPE_NAME),
+    [selection]
+  );
+  const hasNotOnMapContainer = useMemo(() => Boolean(notOnMapContainer), [notOnMapContainer]);
+
+  const notOnMapFeature = useMemo(
+    () => featureTypes.find(feature => feature.typeValue === CONTAINER_NOT_ON_MAP_TYPE_NAME),
+    [featureTypes]
+  );
+
+  const removeContainer = useCallback((itemId: string) => {
+    onChange(selection.filter(({ id }) => id !== itemId));
+  }, [selection, onChange]);
+
+  const removeContainerNotOnMap = useCallback(() => {
+    onChange(selectionOnMap);
+  }, [selectionOnMap, onChange]);
+
+  const updateNotOnMapContainer = useCallback((event: React.ChangeEvent<HTMLInputElement>) => {
+    /* istanbul ignore next */
+    if (notOnMapContainer) {
+      onChange([...selectionOnMap, { ...notOnMapContainer, id: event.currentTarget.value }]);
+    }
+  }, [notOnMapContainer, onChange, selectionOnMap]);
+
+  const addContainerNotOnMap = useCallback(() => {
+    /* istanbul ignore next */
+    if (notOnMapFeature) {
+      onChange([
+        ...selectionOnMap,
+        {
+          id: '',
+          type: notOnMapFeature.typeValue,
+          description: notOnMapFeature.description,
+        },
+      ]);
+    }
+  }, [notOnMapFeature, onChange, selectionOnMap]);
+
+  const toggleNotOnMapContainer = useCallback(({ target: { checked } }: React.ChangeEvent<HTMLInputElement>) => {
+    if (checked) {
+      addContainerNotOnMap();
+    } else {
+      removeContainerNotOnMap();
+    }
+  }, [addContainerNotOnMap, removeContainerNotOnMap]);
+
+  return (
+    <MapPanelContent variant={variant} title="Kies de container" data-testid="selectionPanel">
+      <Paragraph>U kunt meer dan 1 keuze maken</Paragraph>
+
+      {selectionOnMap.length ? (
+        <StyledContainerList selection={selectionOnMap} onRemove={removeContainer} featureTypes={featureTypes} />
+      ) : (
+        <EmptySelectionWrapper>
+          <StyledParagraph>Maak een keuze op de kaart</StyledParagraph>
+        </EmptySelectionWrapper>
+      )}
+
+      <form>
```

Comment:
Is this needed?

Reply:
Not necessarily. I changed it so that the 'enter' key behavior closes the panel without needing a form.